bin/bump.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/bash
+
+  gem install bump
+  git config --global user.name "  Kite Bot"
+  git config --global user.email "kite-bot@heliostech.fr"
+  git remote add authenticated-origin https://kite-bot:$GITHUB_API_KEY@github.com/${DRONE_REPO}
+  git fetch authenticated-origin
+  curl -sL https://deb.nodesource.com/setup_10.x | bash -
```

Comment:
no need to install nodejs every time, only if we pass --js flag to bump.sh

Reply:
actually you take it from barong drone.yml. So maybe @ec will explain why they need nodejs in bump & tag?